csharp/extractor/Semmle.Extraction.CSharp.Standalone/BuildAnalysis.cs

```diff
@@ -173,11 +172,20 @@ private static string ComputeTempDirectory(string srcDir)
         /// </summary>
         void ResolveConflicts()
         {
-            var sortedReferences = usedReferences.
-                Select(r => assemblyCache.GetAssemblyInfo(r.Key)).
-                Where(r => r != AssemblyInfo.Invalid).
-                OrderBy(r => r.Version).
-                ToArray();
+            var sortedReferences = new List<AssemblyInfo>();
+            foreach (var usedReference in usedReferences)
+            {
+                try
+                {
+                    var assemblyInfo = assemblyCache.GetAssemblyInfo(usedReference.Key);
+                    sortedReferences.Add(assemblyInfo);
+                }
+                catch (AssemblyLoadException)
+                {
+                }
```

Comment:
It does feel like we should record this failure somewhere. Why not call `UnresolvedReference`?

Reply:
We can't call `UnresolvedReference` because at this point we're not in a project context, so we wouldn't be able to pass the second parameter. 

At the same time, I'm not sure this catch is required at all. We're iterating over `usedReferences`, which should all be already resolved once, so I don't think this can fail. Also, we had no equality check here previously for the `AssemblyInfo.Invalid` either. 

I'm adding logging to this catch.